docs/contents/components/forms/color-picker/index.mdx

```diff
@@ -274,6 +274,22 @@ To make read-only, set `isReadOnly` to `true`.
 <ColorPicker placeholder="#000000" withResult />
 ```
 
+### Adding Children
```

Comment:
@hironokobari
Most headings use verbs in the imperative form (e.g., "Change Variant", "Set Default Value"). 
To maintain consistency with other headings in the document, could you change the heading to `Add Children`?